nflog/nflogpb/set.go

```diff
@@ -0,0 +1,30 @@
+package nflogpb
```

Comment:
Only reason I'm making this a separate file is so it doesn't get overridden when regenerating from the proto.